lib/openapi/revisions/est-spec-1.0.0-20231214.json

```diff
@@ -0,0 +1,1536 @@
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "The UTC end date and time when usage information is retrieved from (e.g. 2022-01-01T00:00:00Z)",
+            "required": true
+          },
+          {
+            "in": "query",
+            "name": "sortBy",
+            "schema": {
+              "type": "string",
+              "enum": [
+                "Request",
+                "Bytes",
+                "Timestamp"
+              ],
+              "default": "Timestamp"
+            },
+            "description": "Sort by arrtibute",
+            "required": false
+          },
+          {
+            "in": "query",
+            "name": "sortOrder",
+            "schema": {
+              "type": "string",
+              "enum": [
+                "ASC",
+                "DESC"
+              ],
+              "default": "ASC"
+            },
+            "description": "Sort order",
+            "required": false
+          },
+          {
+            "in": "query",
+            "name": "aggregate",
+            "schema": {
+              "type": "boolean",
+              "enum": [
+                true,
+                false
+              ],
+              "default": false
+            },
+            "description": "If the *aggregate* flag is true, then the result are consolidated and returned as an aggregated summary for the specified from and to period.  If the aggregate flag is false or not provided, then only separate records for the same time period is returned.",
+            "required": false
+          },
+          {
+            "in": "query",
+            "name": "limit",
+            "schema": {
+              "type": "number",
+              "default": 20
+            },
+            "description": "The *limit* paramter determines the number of usage records to be returned'"
+          },
+          {
+            "in": "query",
+            "name": "offset",
+            "schema": {
+              "type": "number",
+              "default": 0
+            },
+            "description": "The *offset* from where data to be returned from total quried records"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Usage metrics retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/UsageMetrics"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/variable": {
+      "post": {
+        "tags": [
+          "Variables"
+        ],
+        "description": "Create PhotoniqEST variable",
```

Comment:
```suggestion
        "description": "Create a PhotonIQ EST variable.",
```

Reply:
Need more explanation about what the variable is, what it does, and what the options for body properties are.